app/controllers/application_controller.rb

```diff
@@ -9,6 +9,15 @@ def switch_locale(&action)
     I18n.with_locale(locale, &action)
   end
 
+  def set_time_zone
```

Comment:
We can't put this in ApplicationController -- all our sync controllers in the API inherit from here, and setting our timezone to the reporting timezone could break the timezone conversion that happens when we create records from synced resources.

Reply:
Ohh, this just extracts the method and keeps the `before_action`s in place.  I think this is okay if we rename the action to `set_reporting_time_zone` or something like that...otherwise this almost seems like it should be the default time zone across all actions, which we definitely don't want.

Reply:
Great call on the naming. Hehe your confusion is a good indicator that `set_time_zone` isn't the right name 😅

Reply:
> Renamed to `set_reporting_time_zone` in [78bd51b](https://github.com/simpledotorg/simple-server/commit/78bd51bd95b1ec7701935b2761c200e85335415d)

awesome thanks